plugin.video.filmy/main.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import xbmcgui
+import xbmcplugin
+import requests
+from urlparse import parse_qsl
+
+__url__ = sys.argv[0]
+__handle__ = int(sys.argv[1])
+
+channels = eval(requests.get(url='https://filmy.ml/app/channels.js').text.split("var channels = ")[1].split(";")[0].replace('title', '"title"').replace('description', '"description"').replace('previewImage', '"previewImage"').replace('liveStream', '"liveStream"'))
```

Comment:
You shouldn't run eval blindly out of data provided by a third party, this is extremely unsecure. If this is json, use json.loads()

Reply:
Fixed.